### PR TITLE
feat: add support for  in template vars

### PR DESCRIPTION
**Pull request by cjbell**
### Description

Add docs for `$data` support

### Tasks

KNO-6396

### CODE REVIEW

File: content/designing-workflows/template-editor/overview.mdx [hunk 1]
```diff
@@ -43,11 +43,27 @@ We'll cover:
...
+### Data payload variables
+
+All variables sent in the `data` payload of your workflow trigger call. These don't require a prefixed namespace to reference, so if you send through `{ "a_variable": "something" }` in your data payload you can reference this as `{{ a_variable }}` in your template.
+
+<Callout
+  emoji="💡"
+  text={
+    <>
+      Note: if you want to reference all of the data passed to the workflow, you
+      can use <code>$data</code> variable.
```
Comment (rachael-t) [inline content/designing-workflows/template-editor/overview.mdx:59]:
```suggestion
      can use the <code>$data</code> variable.
```

**Comment by rachael-t** [comment]
Looks like something with the table got thrown off: https://docs-git-cjbell-kno-6396-switchboard-expose-tr-e543fc-knocklabs.vercel.app/designing-workflows/template-editor/variables
Screenshot 2024-07-16 at 7 12 39 AM

File: content/designing-workflows/template-editor/overview.mdx [hunk 2]
```diff
@@ -43,11 +43,27 @@ We'll cover:
...
+    </>
+  }
+/>
+
+### User properties
+
+To reference a user property (such as `name`) use the `actor` and `recipient` namespaces. This looks up the actor or the recipient of a given notification, and then finds the specified property for that user. Here's a code example where a recipient's name and plan are injected into a notification template:
```
Comment (cellomatt) [inline content/designing-workflows/template-editor/overview.mdx:66]: nit

```suggestion
To reference a user property (such as `name`), use the `actor` and `recipient` namespaces. This looks up the actor or the recipient of a given notification, and then finds the specified property for that user. Here's a code example where a recipient's name and plan are injected into a notification template:
```

**Review by cellomatt — APPROVED** [review]
This lgtm once Rachael's changes are addressed

**Pull request merged**